extensions/context-explorer/src/__tests__/acceptance/context-explorer.acceptance.ts

```diff
@@ -67,7 +67,8 @@ describe('Context Explorer (acceptance)', function (this: Mocha.Suite) {
       );
     });
 
-    it('invokes GET /graph', async () => {
+    it('invokes GET /graph', async function (this: Mocha.Context) {
+      if (String(process.platform) === 'os390') return this.skip();
```

Comment:
I think wrapping with `String()` is unnecessary since `process.platform` will always return a string according to the Node.js API Documentation https://nodejs.org/api/process.html#process_process_platform

Reply:
`process.platform` return a Platform object and only allow certain string, so it will have type error, I did try to find where it set the type but couldn't find it so I use the String work around

Reply:
The Platform type is defined by [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) in either types/node/process.d.ts or globals.d.ts depending on the version. We will want to submit a PR to add `'os390'` as a valid value at some point.

For now, I think it would be better to fix the type error using type cast/assert in TypeScript so that the type error will be resolved at compile type, and won't have any runtime effect vs using `String()`.

```suggestion
      if (<string>process.platform === 'os390') return this.skip();
```

or

```suggestion
      if (process.platform as string === 'os390') return this.skip();
```

Reply:
Change made